networking_cisco/plugins/cisco/db/l3/l3_router_appliance_db.py

```diff
@@ -197,7 +199,18 @@ def do_create_router(self, context, router, router_type_id, auto_schedule,
         router_created[routertypeawarescheduler.SHARE_HOST_ATTR] = share_host
         router_created[routerhostingdevice.HOSTING_DEVICE_ATTR] = (
             hosting_device_id)
-        return router_created, r_hd_b_db
+        try:
+            router_db = self._get_router(context, router_created['id'])
+            if gw_info:
+                self._update_router_gw_info(context, router_created['id'],
```

Comment:
In the new code, _update_router_gw_info() is done here. But where its getting called in the old code then?